src/python/pants/bin/local_pants_runner.py

```diff
@@ -271,8 +271,9 @@ def set_start_time(self, start_time):
 
     spec_parser = CmdLineSpecParser(get_buildroot())
     address_specs = [
-      spec_parser.parse_address_spec(spec).to_spec_string()
+      spec_parser.parse_spec(spec).to_spec_string()
       for spec in self._options.specs
+      if isinstance(spec_parser.parse_spec(spec), AddressSpec)
```

Comment:
Does this need to be filtered? It's not clear that it does. The local variable is "address_specs", but I think that's only because everything was renamed to address_specs recently.

Reply:
Yes, because `to_spec_string()` is only defined on `AddressSpec` and not `FilesystemSpec`. So, if this is a `FilesystemSpec`, then we will get an `AttributeError`.

I'm not sure it makes sense for us to yet introduce a superclass `Spec` that has an abstract method `to_spec_string()`. I'd like for us to do a little more discovery on what filesystem specs will look like after all use cases are implemented before generalizing.

Question 1: 
Do we care about reporting the `filesystem_specs` in `RunTracker`? We can tweak this if so.

Question 2:
Do you know why we're converting from the raw address spec into an `AddressSpec`, then going back to the original str? I think we could simply output the original `specs` strings. The only downside I see is that `src/python/pants/util` won't expand to `src/python/pants/util:util`.

Reply:
> Do we care about reporting the filesystem_specs in RunTracker? We can tweak this if so.

IMO, yes: it is a distinct advantage that deprecating `--owner-of` in favor of this feature improves our metrics.